### DISCUSSION TITLE
Answered Questions and duplicates

### DISCUSSION THREAD

**Discussion by GDFaber** [naver/billboard.js#2013] [General]
I'm new to the project, and took a first look at the open issues. There are some entries which have already been answered or fixed in billboard. There are also issues which are already covered by other entries.

I would like to know if issues can be closed in these cases, or if we should rather leave them open (I've been in other Github projects and seen it both ways).

**Comment by netil** [comment]
well, if the issues are definitely looks resolved or fixed, they can be closed.
Some are remaining opened waiting for the issuer check in many cases.

Maybe adopting a bot to deal inactive issue can facilitate our job.
- https://github.com/apps/stale
